x-pack/plugins/canvas/public/state/middleware/es_persist.js

```diff
@@ -31,6 +31,7 @@ export const esPersistMiddleware = ({ getState }) => {
     setWorkpad, // used for loading and creating workpads
     setAssets, // used when loading assets
     resetAssets, // used when creating new workpads
+    setRefreshInterval, // used to set refresh time interval which is a transient value
```

Comment:
While digging around, I also noticed that setRefreshInterval, which doesn't affect persisted data, wasn't in the list of skipped actions in es_persist, so I went ahead and added it.

Reply:
Was this causing a persist call? That list of actions is just a way to shortcut checking for changes in the persistent state. Adding it isn't a problem, just curious if you saw saves happening from this action.

Reply:
No, it didn't make an update call since previous workpad state and new workpad state are the same. I just figured I might as well add it to the list, so we can skip the check.